### PR TITLE
chore(changelogs): publish changelogs for each release

### LINKED ISSUE #465: Publish a consolidated changelog for every release to Workflow docs
Relates to #464 

Good conversation in #community about change visibility. It would be helpful to publish a consolidated changelog for Workflow on our docs site. This should match what we are doing for the release gist: https://gist.github.com/mboersma/247850be1077a3c56754c5d823420d1c#file-v2-4-0-CHANGELOG-md

### PR DESCRIPTION
**Pull request by slack**
Closes #465

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -82,6 +82,13 @@ pages:
     - Controller API v2.0: reference-guide/controller-api/v2.0.md
     - Controller API v2.1: reference-guide/controller-api/v2.1.md
     - Controller API v2.2: reference-guide/controller-api/v2.2.md
+  - Changelogs:
+    - v2.0.0: changelogs/v2.0.0.md
+    - v2.1.0: changelogs/v2.1.0.md
+    - v2.2.0: changelogs/v2.2.0.md
+    - v2.3.0: changelogs/v2.3.0.md
+    - v2.4.0: changelogs/v2.4.0.md
```
Comment (helgi) [inline mkdocs.yml]: misses 2.4.1

Comment (slack) [inline mkdocs.yml]: Yeah, didn't have the changelogs gist anywhere. Will add in a seeeeecond.

Comment (kmala) [inline mkdocs.yml]: can you also add 2.4.2. https://gist.github.com/kmala/3cfeb97c3d2ce2274c908b46ab316898

**Comment by slack** [comment]
Ok. I think I've got everything now... PTAL, especially the release documentation updates. I hope that is clear enough!

**Pull request merged**